File: kin-common/src/test/java/jp/co/kin/common/selenium/LoginTest.java
package jp.co.kin.common.selenium;

import org.junit.Test;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.chrome.ChromeDriver;

import jp.co.kin.common.test.BaseCommonTest;

public class LoginTest extends BaseCommonTest {

	@Test
	public void test() throws InterruptedException {

		System.setProperty("webdriver.chrome.driver",
				"D:\\kin-app\\git\\kin-work\\kin-common\\exe\\chromedriver.exe");

		WebDriver driver = new ChromeDriver();
		driver.get("http://localhost:8080/kin-dashboard/login/index");
		WebElement loginIdElement = driver.findElement(By.id("loginId"));
		loginIdElement.sendKeys("test");
		WebElement passwordElement = driver.findElement(By.id("password"));
		passwordElement.sendKeys("test");
		WebElement submitButton = driver.findElement(By.id("submit"));
		submitButton.click();
		Thread.sleep(10000);

		// これをやると画面が消える
		driver.quit();
	}
}

File: kin-auto/src/main/python/login/LoginTest.py
# -*- coding:utf-8 -*-
'''
Created on 2019/10/20
Login画面→ログイン後のTOP画面のテストを行うpython

@since 1.0.0

'''

from selenium import webdriver

from src.main.python.common import ConstData
from src.main.python.common import Logger
from src.main.python.login import LoginAuth

''' ここからメイン処理 '''

log = Logger.Logger("auto.log")
log.write("LoginTest開始")

# driverを取得
driver = webdriver.Chrome(ConstData.SELENIUM_DRIVER_PATH)

# 正しいログイン情報でログイン
loginId = "test"
password = "test"
LoginAuth.LoginAuth(driver, loginId, password).doLogin()

# ブラウザバック
driver.back()

# 正しくないパスワードでログイン
loginId = "test"
password = "hoge"
LoginAuth.LoginAuth(driver, loginId, password).doLogin()

log.write("LoginTest終了")

''' ここまでメイン処理 '''
